marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/reader/ReadArraysAndMapsTest.java

```diff
@@ -28,8 +30,12 @@ void setup() {
         runAsDataHubOperator();
     }
 
+    /* Running this test on Windows and mac result in SEGFAULT when trying to get partition's row count
+       (readLib.sjs's getRowCountForPartition() method. Once server bug https://bugtrack.marklogic.com/55743
+       is fixed, the test should run fine on all platforms.*/
     @Test
     void arrayOfStrings() {
+        assumeTrue(OS.LINUX.isCurrentOs());
```

Comment:
Can you change this so that instead of a comment, the message is the first argument to assumeTrue? That way, there's no chance that someone later removes this line of code but forgets to remove the comment.